tests/unistd/dup.c

```diff
@@ -20,8 +20,8 @@ int main() {
   errno = 0;
 
   printf("DUP2\n");
-  f = open("/", O_RDONLY);
-  f2 = open("/", O_RDONLY);
+  f = open(".", O_RDONLY);
+  f2 = open(".", O_RDONLY);
```

Comment:
Could we leave a form that checks against `/` in non-NODERAWFS form? I think we'd want to retain that, it's an important invariant of MEMFS that `/` root is predictable.